routes/users.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const knex = require('../database');
+const router = express.Router();
+
+router.route('/:user_id')
+  .get((req, res) => {
+    knex.raw('SELECT * FROM users WHERE id = ?', [req.params.user_id])
+    .then((result) => {
+      if (result.rowCount <= 0){
+        throw { message : 'User not found' };
```

Comment:
Instead of throwing here, just use `res.status(404).json({ message: 'User not found' });`  
This will send back a 404 response with the javascript object as the body. Please apply this pattern through the rest of your code.